xml/art_migration.xml

```diff
@@ -112,6 +112,13 @@
      Guide and Operations Guide.
     </para>
    </listitem>
+   <listitem>
+    <para>
+     &productname; &productnumber; dropped support for internally deployed
+     storage such as Ceph and VSA, move data out of the old internal storage
```

Comment:
This bit doesn't make any sense. Do you mean to say it dropped support BY moving data out of the old internal storage? I'm not sure if this is supposed to be two separate sentences.

Reply:
It means that prior to the migration, customers running HOS need to remove any VSA or  "non SES" Ceph backends deployed by HOS. The data in these should be migrated to a supported backend such as 3PAR or SES. 

The reason for this was customers who may have deployed a newer ceph outside of what was provided by HOS.

Reply:
I tweaked the wording of this a bit, please let me know if it makes more (or less) sense now.